site/pages/getting-started/walkthrough.mdx

```diff
@@ -0,0 +1,91 @@
+---
+outline: deep
+title: Getting started guide
+description: Learn how to get started with Alchemy's Embedded Accounts using Account Kit and the Alchemy Signer, Modular Account, Rundler and Gas Manager.
+---
+
+# Walkthrough
+
+It's time to learn how it all works! Let's take a tour of the [starter app repository](https://github.com/alchemyplatform/embedded-accounts-quickstart) to see how it integrates with embedded accounts.
+
+## 1. The config
+
+First, let's do our configuration in a single file called `config.ts`.
+
+```ts [config.ts]
+// [!include ~/../examples/embedded-accounts-quickstart/src/config.ts:create-accounts-config]
```

Comment:
does the create-accounts-config section render correctly? I can't remember if I had issues with region names that were kebab cased

Reply:
Yup it works! I actually thought it was kebab casing for a minute, but I had to change the region comments in the aa-sdk repo. After that it worked fine ✅